### DISCUSSION TITLE
256

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#701] [Comments]
http://jojoldu.tistory.com/256

**Comment by jojoldu** [comment]
작성자: 박찬욱

좋은 글 잘 보았습니다^^
그런데 EC2에 설치하고 실행하려다 보니 '호스트 몰라요 에러(unknown host)'가 계속 발생해서 카산드라가 계속 내려가는 문제가 있었네요.
/etc/sysconfig/network와 /etc/hosts에 upsource configure에서 baseUrl로 등록하는 public dns를 등록해주면 잘 되더라고요.

저만 그랬는지 모르겠지만.. 혹시나 도움이 될까 싶어 댓글 달아요.

즐거운 명절 되세요~

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

헐 그렇군요;; 제가 설치할때는 발생하지 않아서 전혀 몰랐네요;
댓글 감사합니다!
즐거운 명절 되세요!!

**Comment by jojoldu** [comment]
작성자: sourcereview

안녕하세요 좋은 강의 잘봤습니다
ec2 서버 새로만든 곳에 말씀하신대로 진행하면서  upsource 압축풀어서 base url 지정한거까지 세팅돼서
~~ 
* Configuring Configuration Wizard
까지 정상적으로 떳고,
sudo bin/upsource.sh start
날리니까

~~
 JetBrains Upsource 2017.3 Configuration Wizard will be available on [public DNS] after start 
Upsource is running
 
이렇게 나오더라구요 
url 접속하니까 계속 접속이 안돼서
다시 한 번 더 스타트 하니까 "Upsource is already running"
이렇게 나오는데 여전히 접속이 안되네요ㅜㅜ
혹시 예상되는 문제나, 로그기록같은거 확인할 방법이 있을까요??ㅜㅜ

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

안녕하세요? 저는 만나지 못해서 확실한 답변은 못드립니다만 ㅠ
로그는
/home/ec2-user/upsource/upsource-2017.3.2593 (업소스 버전에 따라 다름)/logs
에 가시면 확인하실수 있습니다!

**Comment by chowon104** [comment]
JetBrains Upsource 2017.3 Configuration Wizard will be available on [public DNS] after start
Upsource is running

저도 이 에러가 발생했는데 이거는 AWS의 보안 그룹에 8080 포트 추가해주면 해결됩니다 :)

**Comment by jojoldu** [comment]
@hY43 앗 대신 답변해주셔서 고맙습니다^^

**Comment by chowon104** [comment]
@jojoldu 네넵ㅎㅎ 저.. 혹시 질문이 있는데 EC2 내에 별도의 웹 서버를 올리지 않아도 사용이 가능한가요?? 처음에 setup에서 설정 입력하고 연결하는 중에 카산드라가 올라가다가 멈춰서요..ㅎㅎ

**Comment by jojoldu** [comment]
@hY43 upsource가 자체 웹서버를 구동시키기 때문에 upsource를 실행시키시면 별도의 웹 서버를 구동시키실 필요는 없습니다^^
혹시나 ec2주소:8080 으로 upsource가 접속안되시면 맨윗 댓글의 박찬욱님이 남기신것처럼
/etc/sysconfig/network와 /etc/hosts에 upsource configure에서 baseUrl로 등록하는 public dns를 등록해주시면 될것같습니다^^

**Comment by cheese10yun** [comment]
좋은 자료 감사합니다. 10 이하 팀에게는 무료인점이 정말 좋네요

**Comment by yangseungmi** [comment]
안녕하세요
발표듣고 Upsource 찾아보다가 블로그 글도 잘보고 갑니다~

**Comment by jojoldu** [comment]
@yangseungmi 앗 발표 보셨던 분이시군요!
발표도 봐주시고 블로그도 봐주셔서 감사합니다 :)

**Comment by hsoh-neibus** [comment]
smpt 세팅 하는 부분에서 아무 정보나 넣고서 gmail로 보내면 sendmail 로그에 stat=Service unavailable 이렇게 표시 되는데, 동욱님은 어떻게 저렇게 잘 넘어 가는건가요?
찾아보니 gmail 같이 큰 메일 서비스 업체로 메일을 보내려면 통과해야 하는 인증 절차들이 많이 있던데.. 꿀팁좀 있으면 알려주세요..

**Comment by imdudu1** [comment]
감사합니다 ㅎㅎ

**Comment by cyr9210** [comment]
aws 프리티어 내에서 설치 성공하신분 있나요?

저는 ec2 용량때문인지 오류가 계속 발생하네요. (30GB로 설정)
